### PR TITLE
[OTA-3289] In Device Groups, the devices cannot be dragged out properly

### PR DESCRIPTION
**Pull request by piotrcichonmee**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/devices/Item.jsx [hunk 1]
```diff
@@ -14,14 +14,20 @@ import { DEVICE_STATUS_ERROR, DEVICE_STATUS_OUTDATED, DEVICE_STATUS_UP_TO_DATE }
...
+    const { selectedGroup } = groupsStore;
     const { uuid } = props.device;
     const foundClassicGroup = _.find(
       groupsStore.groups, group => group.devices.values.indexOf(uuid) > -1 && group.groupType === 'static'
     );
-
+    let groupId;
+    if (foundClassicGroup) {
+      groupId = foundClassicGroup.id;
+    } else if (!selectedGroup.isSmart && selectedGroup.type === 'real') {
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/components/devices/Item.jsx]: 'real' to constants

File: ota-plus-web/app/reactapp/src/components/devices/Item.jsx [hunk 2]
```diff
@@ -14,14 +14,20 @@ import { DEVICE_STATUS_ERROR, DEVICE_STATUS_OUTDATED, DEVICE_STATUS_UP_TO_DATE }
 const deviceSource = {
   beginDrag(props) {
     const { groupsStore } = props.stores;
+    const { selectedGroup } = groupsStore;
     const { uuid } = props.device;
     const foundClassicGroup = _.find(
       groupsStore.groups, group => group.devices.values.indexOf(uuid) > -1 && group.groupType === 'static'
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/components/devices/Item.jsx]: 'static' to constants

File: ota-plus-web/app/reactapp/src/components/devices/Item.jsx (hunk 1 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/devices/Item.jsx]: Done

File: ota-plus-web/app/reactapp/src/components/devices/Item.jsx (hunk 2 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/devices/Item.jsx]: Done

**Pull request merged**